src/main/java/gov/cabinetofice/gapuserservice/enums/LoginJourneyState.java

```diff
@@ -73,6 +73,7 @@ public LoginJourneyRedirect getLoginJourneyRedirect(final RoleEnum role) {
                 case SUPER_ADMIN -> LoginJourneyRedirect.SUPER_ADMIN_DASHBOARD;
                 case ADMIN -> LoginJourneyRedirect.ADMIN_DASHBOARD_MIGRATION_PASS;
                 case APPLICANT, FIND -> LoginJourneyRedirect.APPLICANT_APP_MIGRATION_PASS;
+                case TECHNICAL_SUPPORT ->  LoginJourneyRedirect.TECHNICAL_SUPPORT_DASHBOARD;
```

Comment:
if we check this last it means people with the technical support role will have to have only this role in order to ever actually be directed to their dashboard. this means they won't be able to access any other part of the application (find, apply, admin) - is this what we want?

Reply:
Will double check before merging. I've heard conflicting ideas about what is desired here. Cheers for pointing out Jamie!